modules/EnsEMBL/Web/Command/Help/SendEmail.pm

```diff
@@ -34,16 +34,17 @@ sub process {
   my $hub     = $self->hub;
   my $url;
 
-  if ($hub->param('logic') != '9') {
-    throw exception('Form validation error', 'Answer to the logical question is wrong');
-  }
+  my $logic_value = $hub->param('logic');
+  $logic_value =~ s/^\s+|\s+$//g;
 
-  if ($hub->param('submit') eq 'Back') {
+  my $error_msg = $logic_value != 9 ? "Are you a human verification failed! Please try again." : "";
```

Comment:
"Are you a human verification failed" reads awkward. Perhaps, "You did not provide the correct response to the control question. Please try again."?

Reply:
response to the logical question? instead of control?

Reply:
"You did not provide the correct response to the logical question" doesn't sound right either 😞